internal/cloudapi/v2/imagerequest.go

```diff
@@ -464,6 +464,9 @@ func getTarget(targetType UploadTypes, options UploadOptions, request *ComposeRe
 
 	case UploadTypesLocal:
 		irTarget = target.NewWorkerServerTarget()
+		irTarget.ImageName = imageType.Filename()
+		irTarget.OsbuildArtifact.ExportFilename = imageType.Filename()
+		irTarget.OsbuildArtifact.ExportName = imageType.Exports()[0]
```

Comment:
This assignment is already handled on line no. 476, so it should not be here. And it would make sense to move there also the `irTarget.OsbuildArtifact.ExportFilename = imageType.Filename()` line, because all of the `new*Target()` functions do the same thing.